src/includes/getting-started-primer/apple.macos.mdx

```diff
@@ -12,7 +12,7 @@
 - Events [enriched](/platforms/apple/enriching-events/context/) with device data
 - Offline caching when a device is unable to connect; we send a report once we receive another event
 - [Breadcrumbs automatically](/platforms/apple/enriching-events/breadcrumbs/#automatic-breadcrumbs) captured for
-  - Application lifecycle events (`didBecomeActive`, `didEnterBackground`, `viewDidAppear`)
```

Comment:
was it a mistake or intentional? seems non related to the PR

Reply:
The PR previously had more changes on this file. I didn't want to open up a PR for this small change. So yes, it was intentional.